src/tests/suite/daos_test_common.c

```diff
@@ -492,8 +492,10 @@ pool_destroy_safe(test_arg_t *arg, struct test_pool *extpool)
 			return rc;
 		}
 
+		print_message("waiting for rebuild, ver %u, sec %u, err %d, state %d,"
+			      "fail_rank %d\n", rstat->rs_version, rstat->rs_seconds,
+			      rstat->rs_errno, rstat->rs_state, rstat->rs_fail_rank);
 		if (rstat->rs_state == DRS_IN_PROGRESS) {
-			print_message("waiting for rebuild\n");
```

Comment:
i'd rather you keep that print in the loop (which was easily used to identify that the test was stuck here actually) and especially since there is a sleep there.

Reply:
was not requesting changes for this PR, but to follow on

Reply:
There's still a print in the loop, it's just outside the conditional and contains more information now.

Reply:
sorry, i have not fully woken up yet.. not sure why i thought it was move out :-/